src/utils/components/PolicyForm/PolicyInterfaceExpandable.tsx

```diff
@@ -52,48 +52,58 @@ const PolicyInterfacesExpandable: FC<PolicyInterfacesExpandableProps> = ({
 
   return (
     <>
-      {policy?.spec?.desiredState?.interfaces.map((policyInterface, index) => (
-        <FormFieldGroupExpandable
-          key={`${policyInterface.type}-${policyInterface?.name}`}
-          className="policy-interface__expandable"
-          toggleAriaLabel={t('Details')}
-          isExpanded={true}
-          header={
-            <FormFieldGroupHeader
-              titleText={{
-                text: getExpandableTitle(policyInterface, t),
-                id: `nncp-interface-${index}`,
-              }}
-              actions={
-                <Tooltip content={t('Remove interface')}>
-                  <Button
-                    variant="plain"
-                    aria-label={t('Remove')}
-                    onClick={() => removeInterface(index)}
+      {policy?.spec?.desiredState?.interfaces.map((policyInterface, index) => {
+        const createdInterface = createdInterfacesNames?.includes(policyInterface?.name);
```

Comment:
I suggest making this `interfaceCreated`. `createdInterface` sounds like I'm getting the interface that was created.